generators/lua/math.js

```diff
@@ -127,14 +127,62 @@ Blockly.Lua['math_constant'] = function(block) {
   return CONSTANTS[block.getFieldValue('CONSTANT')];
 };
 
+Blockly.Lua['math_constant'] = function(block) {
+  // Constants: PI, E, the Golden Ratio, sqrt(2), 1/sqrt(2), INFINITY.
+  var CONSTANTS = {
+    PI: ['math.pi', Blockly.Lua.ORDER_HIGH],
+    E: ['math.exp(1)', Blockly.Lua.ORDER_HIGH],
+    GOLDEN_RATIO: ['(1 + math.sqrt(5)) / 2', Blockly.Lua.ORDER_MULTIPLICATIVE],
+    SQRT2: ['math.sqrt(2)', Blockly.Lua.ORDER_HIGH],
+    SQRT1_2: ['math.sqrt(1 / 2)', Blockly.Lua.ORDER_HIGH],
+    INFINITY: ['math.huge', Blockly.Lua.ORDER_HIGH]
+  };
+  return CONSTANTS[block.getFieldValue('CONSTANT')];
+};
+
 Blockly.Lua['math_number_property'] = function(block) {
   // Check if a number is even, odd, prime, whole, positive, or negative
   // or if it is divisible by certain number. Returns true or false.
-  const number_to_check = Blockly.Lua.valueToCode(block, 'NUMBER_TO_CHECK',
-      Blockly.Lua.ORDER_MULTIPLICATIVE) || '0';
-  const dropdown_property = block.getFieldValue('PROPERTY');
+  const PROPERTIES = {
+
```

Comment:
Remove blank lines as per the style guide. https://developers.google.com/blockly/guides/modify/web/style-guide